pkg/test/kube/accessor.go

```diff
@@ -365,6 +365,7 @@ func (a *Accessor) CreateNamespace(ns string, istioTestingAnnotation string, inj
 	}
 	if injectionEnabled {
 		n.ObjectMeta.Labels["istio-injection"] = "enabled"
+		n.ObjectMeta.Labels["istio-env"] = "istio-control"
```

Comment:
Once https://github.com/istio/istio/pull/13830 is merged let's update this to be based on the passed in flag

Reply:
It got merged, can we use the controlNamespace arg here?